apollo-normalized-cache-api/src/commonTest/kotlin/com/apollographql/apollo3/cache/normalized/CacheKeyResolverTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.apollographql.apollo3.cache.normalized
+
+import com.apollographql.apollo3.api.CompiledField
+import com.apollographql.apollo3.api.CompiledListType
+import com.apollographql.apollo3.api.Executable
+import com.apollographql.apollo3.api.ObjectType
+import com.apollographql.apollo3.cache.normalized.CacheKeyResolverTest.Fixtures.TEST_LIST_FIELD
+import com.apollographql.apollo3.cache.normalized.CacheKeyResolverTest.Fixtures.TEST_SIMPLE_FIELD
+import com.apollographql.apollo3.exception.CacheMissException
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+
+class CacheKeyResolverTest {
+
+  lateinit var subject: CacheKeyResolver
+  lateinit var onCacheKeyForField: (field: CompiledField, variables: Executable.Variables) -> CacheKey?
+  lateinit var onListOfCacheKeysForField: (field: CompiledField, variables: Executable.Variables) -> List<CacheKey?>?
+
+  @BeforeTest
+  fun setup() {
+    subject = FakeCacheKeyResolver()
+    onCacheKeyForField = { _, _ ->
+      TODO("Unexpected call to cacheKeyForField")
+    }
+    onListOfCacheKeysForField = { _, _, ->
+      TODO("Unexpected call to listOfCacheKeysForField")
+    }
+  }
+
+  @Test
+  fun `cacheKeyForField called for named composite field`() {
```

Comment:
This fails in CI with 
```
Name contains illegal chars that can't appear in JavaScript identifier
```

I guess Javascript isn't too happy about spaces in method names, same for other methods bellow. Making everything `camelCase` or `snake_case` should fix the tests